3rd_party/prereq-build.sh

```diff
@@ -26,6 +26,9 @@ echo "Building protobuf"
     CFLAGS="`${PYTHON_CONFIG} --cflags` -fPIC" LDFLAGS="`${PYTHON_CONFIG} --ldflags`" ./configure "--prefix=$TARGET/protobufkit"
     make
     make install
+    cd "./python"
+    python setup.py build
+    sudo python setup.py install
```

Comment:
This isn't right, don't install anything with `sudo` rights and don't install it globally. End user might not want to have this version of protobuf as a system package (or might not even have `sudo` rights).

Reply:
removed `sudo python setup.py install`